index.rst

```diff
@@ -260,92 +291,98 @@ If working with an individual CSC, which as an operator would be a rare occurren
+#.  Edit/Add/Replace the configuration file(s) in the CSC's configuration directory.
 
-        # Labels for recommended configurations; a dictionary of {label: config_file}
-        default: _base.yaml
-        hex_m1: hex_m1_hex_202003.yaml
+        - If editing the ``_init.yaml`` or a ``_<site>.yaml`` file, the filename must remain unchanged.
+        - If editing or adding an configuration override file, ideally the name of the file should reflect the purpose of change; dates can also be used as well.
+          Old configuration files can be kept in the repo if they still represent valid configurations. Otherwise, they should be removed.
+          Note, though, that they will still remain available on previous commits in the git repo, enabling historical comparison.
 
-#.  Add, commit and push the changes, with a commit message.
+#.  Fill out the required metadata at the top of the file detailing where any auxiliary data may be stored, the Jira ticket number used to create the file, and the reason for creating the configuration, such as in `this example <https://tstn-017.lsst.io/v/PREOPS-27/_downloads/ATSpectrograph_example_config.yaml>`__.
 
-    ::
 
-      git commit -am "Updated base configuration to use the configuration previously labeled constants_hex with a filename of hex_m1_hex_202003.yaml. Added hex_m1_hex_202003.yaml with label of hex_ml. See DM-12345 for more information."
-      git push
+#.  If you have an environment to do so, such as the standard T&S development container, run the unit tests in the package locally.
 
-    The commit message can add information about what changes are being made and a short description for the reason.
-    It is also recommended to explicitly mention the Jira ticket for the work being done as the branch name is lost once the changes are merged to the head branch.
+#.  Add, commit and push the changes, with a commit message.
 
-#.  If this is a normal configuration change procedure, then create a pull-request (PR), and have it reviewed, merged and released.
+    .. prompt:: bash
 
-    .. TODO: Fix/Edit/Verify the example below to checkout a local version of
-    .. the repo, then set it up accordingly.
+        git commit -am "Add new LUTs for ATAOS (file 20200512-configuration.yaml) based on data taken on 20200512. Check DM-12345 for more information."
+        git push
 
+#. Verify the continuous integration tests pass. If they don't, fix the issue and repeat the previous step.
 
-#.  Once the new configuration is released it can be made available to the component, which will not automatically see the newly created configuration.
-    During normal operations this involves creating a new deployable artifact and updating the deployment to use the new configuration version.
+#.  Test the new configuration on the CSC.
+    If this requires in-dome or on-sky testing, then create an annotated alpha release tag. Then make sure the test is properly documented in a technote and/or Jira ticket.
```

Comment:
alpha or beta?

Reply:
Unsure... isn't it alpha if it's untested? 
People are going to botch the versioning constantly....